src/providers/retryProvider.ts

```diff
@@ -199,21 +205,31 @@ export class RetryProvider extends ethers.providers.StaticJsonRpcProvider {
       .filter(([, result]) => compareRpcResults(method, result, quorumResult))
       .map(([provider]) => provider.connection.url);
     if (Object.keys(mismatchedProviders).length > 0 || errors.length > 0) {
-      logger.warn({
-        at: "ProviderUtils",
-        message: "Some providers mismatched with the quorum result or failed 🚸",
-        notificationPath: "across-warn",
-        method,
-        params,
-        quorumProviders,
-        mismatchedProviders,
-        erroringProviders: errors.map(([provider, errorText]) => formatProviderError(provider, errorText)),
-      });
+      this._logQuorumMismatchOrFailureDetails(method, params, quorumProviders, mismatchedProviders, errors);
     }
 
     return quorumResult;
   }
 
+  _logQuorumMismatchOrFailureDetails(
+    method: string,
+    params: Array<unknown>,
+    quorumProviders: string[],
+    mismatchedProviders: { [k: string]: unknown },
+    errors: [ethers.providers.StaticJsonRpcProvider, string][]
+  ) {
+    logger.warn({
+      at: "ProviderUtils",
+      message: "Some providers mismatched with the quorum result or failed 🚸",
+      notificationPath: "across-warn",
+      method,
+      params: JSON.stringify(params),
+      quorumProviders,
+      mismatchedProviders: JSON.stringify(mismatchedProviders),
+      erroringProviders: errors.map(([provider, errorText]) => formatProviderError(provider, errorText)),
+    });
+  }
+
```

Comment:
IMO this one could probably also be a closure, since as far as I can see, we don't really expect to call it from any other methods.

Reply:
Yes, you're right. I think we'll be calling it only from this method. I modified it here [3165d0d](https://github.com/across-protocol/sdk/pull/728/commits/3165d0d36a2964c1382bca6af4838458cba3cb2f)